backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -369,6 +383,8 @@ def set_user_password(self, context, request: TracimRequest, hapic_data=None):
         uapi = UserApi(
             current_user=request.current_user, session=request.dbsession, config=app_config  # User
         )
+        if not has_write_rights("password", app_config, request):
+            return HTTPBadRequest()
```

Comment:
You must return some explanation. Error cases must be conform to Tracim error code strategy: You must return a Tracim error code + details. See `error.py` + `class ErrorCode`

Reply:
@lebouquetin Could you give more explanation about the *how* please ? I can't find usage in the code, all I get is `raise xxx` which triggers a HTTP 500, followed by an "unexpected error" in the frontend.

If I return a `HTTPBadRequest()` with error code + details in the body, frontend still displays a generic `Error while changing username` error.

If providing more details in the code and in the request body, then I know how to do it, but to send more info to the frontend, I not sure of how...